src/main/java/Truck.java

```diff
@@ -0,0 +1,12 @@
+public class Truck extends Machine {
+    @Override
+    public void doWork() {
+        System.out.println("Truck is working now!");
+    }
+
+    @Override
+    public void stopWork() {
+        System.out.println("Truck stops working!");
+    }
+

```

Comment:
empty line

Reply:
have deleted it and created new package for all classes

Reply:
ready to watch